lib/Weasel/Driver/Selenium2.pm

```diff
@@ -297,7 +297,12 @@ sub execute_script {
 sub get_attribute {
     my ($self, $id, $att) = @_;
 
-    return $self->_resolve_id($id)->get_attribute($att,1);
+    my $element = $self->_resolve_id($id);
+    my $value;
+    $value = $element ->get_attribute($att)  # Try with property/attribute
+        if $self->_driver->{is_wd3};
+    return $value
+        // $element ->get_attribute($att,1); # Force using attribute
```

Comment:
There's an extra space in this line, before the "dereferencing arrow" (same 3 lines up).